PredicateKit/CoreData/NSManagedObjectContextExtensions.swift

```diff
@@ -370,6 +370,28 @@ public struct FetchRequest<Entity: NSManagedObject> {
     let request: NSFetchRequest<Entity> = requestBuilder.makeRequest(from: self)
     return try context.count(for: request)
   }
+  
+   /// An NSFetchedResultsController initialized with the fetch request.
+   ///
+   /// - Parameters:
+   ///    - sectionNameKeyPath - keypath on resulting objects that returns the section name. This will be used to pre-compute the section information. Defaults to `nil`.
```

Comment:
```suggestion
   ///    - sectionNameKeyPath - A  key path on resulting objects that returns the section name. This will be used to pre-compute the section information. Defaults to `nil`.
```